src/registrar/models/user.py

```diff
@@ -163,6 +163,16 @@ def get_active_requests_count(self):
         active_requests_count = self.domain_requests_created.filter(status__in=allowed_states).count()
         return active_requests_count
 
+    def get_expiring_domains(self, request):
+        """Return boolean if expiring/expired domains exists"""
+        domain_ids = self.get_user_domain_ids(request)
+        domains = Domain.objects.filter(id__in=domain_ids)
+        how_many_expired_domains = 0
```

Comment:
Is there a way to get the length of domains that are expiring here with a filter rather than iterating through every domain and then returning the length of that filtered list of expiring domains?

Reply:
If the filter approach didn't work, I suggest renaming this variable to num_expired_domains

Reply:
@erinysong I didn't quite find an alternative approach to using the filer. The best I can do is make it one line. Would that be cleaner?

Reply:
It would still be a for loop with individual checking each domain

Reply:
Got it, thank you - looks like Django only does filtering on model attributes and not on methods so that makes sense the filter wouldn't work on the is_expiring method.

Reply:
I think in that case we can keep the for loop to iterate. The other option would be to set a filter that directly checks again if the expiration date is our window but that also has the caveat of repeating logic

Reply:
@zandercymatics thank you for your input! I couldn't figure it out. I have updated it to count via filter